### ISSUE TITLE
Cannot start screeps server

### ISSUE THREAD

**Issue by compilenix** [screeps/screeps#83]
Do you have any ideas for me?

i'm able to login but the server seems to have been paused, no errors or something like that (except the connection reset in the `storage.log`.

This problem persists even if i remove the whole directory `/home/screeps/world` and start over.

Ubuntu 18.10
NodeJS 10.5.0 (via nvm)

```sh
su -l screeps
cd /home/screeps/world
npm install screeps
npx screeps init
npx screeps start
```

```
screeps@screeps1:~/world$ npx screeps start
Server version 3.0.0
Starting all processes. Ports: 21025 (game), 21026 (cli)
[storage] process 24666 started
[backend] process 24676 started
[engine_main] process 24677 started
[engine_runner] process 24682 started
[engine_processor1] process 24687 started
[engine_processor2] process 24692 started
[engine_processor3] process 24698 started
[engine_processor4] process 24704 started
[engine_processor5] process 24710 started
[engine_processor6] process 24717 started
[engine_processor7] process 24725 started
[engine_processor8] process 24730 started
[engine_runner] process 24682 exited with code null, see 'logs' folder for more details
[engine_runner] process 24790 started
[engine_runner] process 24790 exited with code null, see 'logs' folder for more details
[engine_runner] process 24804 started
[engine_runner] process 24804 exited with code null, see 'logs' folder for more details
[engine_runner] process 24816 started
[engine_runner] process 24816 exited with code null, see 'logs' folder for more details
[engine_runner] process 24828 started
```

engine_runner.log
```
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Terrain shared buffer size: 357500
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Terrain shared buffer size: 357500
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Terrain shared buffer size: 357500
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Terrain shared buffer size: 357500
```

backend.log
```
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Starting CLI server
STEAM_KEY environment variable found, disabling native authentication
Connecting to Steam Web API
CLI listening on localhost:21026
Starting game server (protocol version 13)
SockJS v0.3.19 bound to "/socket"
Game server listening on 0.0.0.0:21025
Server password is xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
GET /socket/info 5ms 200
GET /socket/info 1ms 200
GET /socket/941/c0e9lzdr/websocket 4ms (unfinished)
GET /socket/361/m6_mkffu/websocket 1ms (unfinished)
Running cronjob 'sendNotifications'
Running cronjob 'roomsForceUpdate'
Running cronjob 'genPowerBanks'
Running cronjob 'genInvaders'
Running cronjob 'purgeTransactions'
Running cronjob 'recreateNpcOrders'
Running cronjob 'calcMarketStats'
```

engine_main.log
```
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
Main loop reset! Stage: waitForUsers
Game time set to 185
Main loop reset! Stage: waitForUsers
Game time set to 186
Main loop reset! Stage: waitForUsers
Game time set to 187
Main loop reset! Stage: waitForUsers
```

engine_processor1.log (the other engine processors have the exact same log file contents)
```
Loading mods from "/home/screeps/world/mods.json"
Connecting to storage
```

storage.log
```
Loading mods from "/home/screeps/world/mods.json"
Starting storage server
Storage listening on 21027
[127.0.0.1:57832] Incoming connection
[127.0.0.1:57834] Incoming connection
[127.0.0.1:57836] Incoming connection
[127.0.0.1:57838] Incoming connection
[127.0.0.1:57840] Incoming connection
[127.0.0.1:57842] Incoming connection
[127.0.0.1:57844] Incoming connection
[127.0.0.1:57846] Incoming connection
[127.0.0.1:57848] Incoming connection
[127.0.0.1:57850] Incoming connection
[127.0.0.1:57852] Incoming connection
[127.0.0.1:57842] Connection closed
[127.0.0.1:57856] Incoming connection
[127.0.0.1:57856] Connection error: read ECONNRESET
[127.0.0.1:57856] Connection closed
[127.0.0.1:57858] Incoming connection
(node:26573) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 add listeners added. Use emitter.setMaxListeners() to increase limit
[127.0.0.1:57858] Connection closed
[127.0.0.1:57860] Incoming connection
[127.0.0.1:57860] Connection error: read ECONNRESET
[127.0.0.1:57860] Connection closed
[127.0.0.1:57864] Incoming connection
[127.0.0.1:57864] Connection closed
[127.0.0.1:57866] Incoming connection
(node:26573) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 add listeners added. Use emitter.setMaxListeners() to increase limit
[127.0.0.1:57866] Connection closed
[127.0.0.1:57868] Incoming connection
[127.0.0.1:57868] Connection closed
[127.0.0.1:57870] Incoming connection
[127.0.0.1:57870] Connection error: read ECONNRESET
[127.0.0.1:57870] Connection closed
[127.0.0.1:57872] Incoming connection
(node:26573) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 add listeners added. Use emitter.setMaxListeners() to increase limit
```

**Comment by siasur** [comment]
I have the same problem :(

**Comment by compilenix** [comment]
I resolved the problem be downgrading to NodeJS 9.11.2
